app/src/main/java/com/example/notes2text/adapters/AccountSettingsAdaptors/PopUpUsernamePresenter.java

```diff
@@ -0,0 +1,26 @@
+package com.example.notes2text.adapters.AccountSettingsAdaptors;
+
+import android.widget.EditText;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.notes2text.usecases.UserUpdateInfo;
+
+public class PopUpUsernamePresenter extends AppCompatActivity {
+
+    public PopUpUsernamePresenter(){}
```

Comment:
for all controllers and presenters, consider removing the constructor for ease of view as it is the same as Java default constructor.